infrastructures/docker/infrastructure/src/main/java/org/eclipse/che/workspace/infrastructure/docker/environment/DockerEnvironmentNormalizer.java

```diff
@@ -40,11 +47,20 @@ public void normalize(DockerEnvironment dockerEnvironment, RuntimeIdentity ident
     Map<String, DockerContainerConfig> containers = dockerEnvironment.getContainers();
     for (Map.Entry<String, DockerContainerConfig> containerEntry : containers.entrySet()) {
       DockerContainerConfig containerConfig = containerEntry.getValue();
+
+      String userName;
+      try {
+        User user = userDao.getById(identity.getOwnerId());
+        userName = user == null ? identity.getOwnerId() : user.getName();
+      } catch (NotFoundException | ServerException e) {
+        userName = identity.getOwnerId();
+      }
+
       containerConfig.setContainerName(
           containerNameGenerator.generateContainerName(
               identity.getWorkspaceId(),
               containerConfig.getId(),
-              identity.getOwnerName(),
+              userName,
```

Comment:
I'd say that using owner id here is even better. @garagatyi WDYT?

Reply:
so the container naming strategy will change ? Doesn't it have any impact on UI for the end-user ?

Reply:
I think no. The only one person who sees it is Che Admin using docker CLI. @garagatyi Am I right?

Reply:
I agree that container name is the only place. And it doesn't make much sense to keep username there if it costs us something.